ChunkyDocs/docs/getting_started/installing_chunky.md

```diff
@@ -0,0 +1,162 @@
+
+---
+
+## Setup
+
+- Step 1: Download the <a href="https://adoptium.net/temurin/releases" target="_blank"><strong>Java 17 JRE</strong></a> for your platform.[^3]
+
+- Step 2: Download the <a href="https://gluonhq.com/products/javafx" target="_blank"><strong>OpenJFX 18.0.2 SDK</strong></a> for your platform.[^3] [^4] OpenJFX is not required to run Chunky [headlessly](../../reference/user_interface/headless) (via command line).
+
+- Step 3: Download the <a href="https://chunkyupdate.lemaik.de/ChunkyLauncher.jar" target="_blank"><strong>Chunky Launcher (Universal JAR)</strong></a> and save it to a safe place on your computer (you will use this to start Chunky).
+
+Further setup instructions for [Windows](#windows), [Linux](#linux), and [macOS](#macos) are located below.
+
+---
+
+### Windows
+
+- Step 4: If you downloaded the Java 17 installer, then run it to install Java 17 on your computer. If you downloaded the Java 17 ZIP archive, then extract it to a safe place on your computer.
+ 
+- Step 5: Extract from the OpenJFX ZIP archive the `bin`, `legal`, and `lib` folders to `C:\Program Files\openjfx` or `C:\Users\<username>\.chunky\javafx`, creating these folders if necessary.
+ 
+- Step 6: Start the ChunkyLauncher.jar. This can usually be done by double-clicking it, although you may need to start it via a command line or script using the command, `java -jar "<path\to\ChunkyLauncher.jar>" --launcher`.[^6] [^7] This is required if you downloaded the Java 17 ZIP archive, unless you manually properly set JAR files to open with Java 17, in which case you can start the Chunky Launcher by double-clicking it. If JAR files are not properly set to open with Java 17, then the command to start it is, `"<path\to\Java 17\java.exe>" -jar "<path\to\ChunkyLauncher.jar>" --launcher`.[^6] [^7]
+
+<div class="video-wrap">
+  <div class="video-container">
+	<iframe src="https://www.youtube.com/embed/GTUhZVjatPY" allowfullscreen="true"></iframe>
+  </div>
+</div>
+
+---
+
+### Linux
+
+- Step 4: Install Java 17 on your computer.
+
+- Step 5: Extract from the OpenJFX ZIP archive the `bin`, `legal`, and `lib` folders to `/usr/share/openjfx` or `~/.chunky/javafx`, creating these folders if necessary.
+
+- Step 6: Start the ChunkyLauncher.jar with the command, `"</path/to/Java 17/java>" --module-path "~/.chunky/javafx/lib" --add-modules javafx.controls,javafx.fxml -jar "</path/to/ChunkyLauncher.jar>" --launcher`.[^6] [^7]
+
+---
+
+### macOS
+
+On M1-equipped macs, which are aarch64 (ARM-based), Rosetta 2 enables an emulation, of sorts, of x64 macOS applications. Please ensure that both the JRE and OpenJFX have matching architectures. We recommended native aarch64; however, x64 performance should be similar. Furthermore, there are many other issues and bugs that are difficult to resolve. Please ask for assistance and we will try our best.
```

Comment:
To my knowledge there is no indication to the end user that this layer is used unless you dig for it.

Maybe we should relocate it to troubleshooting or something? While it is outside of the scope of Chunky itself it can still have an impact on the user and it's more food for thought in that sense.